### PR TITLE
Document updating table behavior.

### PR DESCRIPTION
**Pull request by jacksonrnewhouse**

### CODE REVIEW

File: sql/updating-tables.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Updating Tables
+description: 'Semantics for updating tables in Arroyo SQL'
+---
+
+In addition to handling real-time data using windows and watermarks, 
+Arroyo SQL also supports updating tables. This is done by splitting rows into _inserts_ and _deletes_.
+Inserts are new rows that have been added to the table, while deletes are rows that have been removed.
+
+#### Reading from Updating Sources
```
Comment (mwylde) [inline sql/updating-tables.mdx:10]: Nit: we tend to only capitalize the first word of headers in our docs

File: sql/updating-tables.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+#### Writing to Updating Sinks
+Debezium data can be written to sinks with the `debezium_json` format. 
+This will convert inserts into a `'c'` (create) operation, deletes into a `'d'` (delete) operation.
+
+#### Non-windowed Aggregates
+Aggregating data without a window will result in an updating output. 
+This will emit an insert the first time data is processed for a group and subsequent data 
+will retract the prior value and then insert the new value. 
+Aggregates are buffered in the operator, emitting every second of wallclock time.
```
Comment (mwylde) [inline sql/updating-tables.mdx:27]: Did we make this configurable via environment variable?

Comment (jacksonrnewhouse) [reply sql/updating-tables.mdx:27]: No

Comment (jacksonrnewhouse) [reply sql/updating-tables.mdx:27]: Working on that now

**Pull request merged**